Makefile

```diff
@@ -3,7 +3,8 @@
 SUBDIRS = source/man
 
 # You can set these variables from the command line.
-SPHINXOPTS    += -W
+O            ?= -t man
```

Comment:
commit message is wrong

Reply:
still, this is not adding any "shortcut target". 

Commit message could be something like
```
Makefile: Build core manpages by default
```

Reply:
git interactive rebase fooled me again :)